contracts/base/DAIPool.sol

```diff
@@ -0,0 +1,139 @@
+/*
+    Copyright 2020 Fabrx Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+pragma solidity 0.5.17;
+
+// Libraries
+import "../util/AddressLib.sol";
+
+// Commons
+import "../util/ZeroCollateralCommon.sol";
+
+// Interfaces
+import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import "../interfaces/DAIPoolInterface.sol";
+import "../interfaces/LenderInfoInterface.sol";
+import "../interfaces/LoansInterface.sol";
+import "../interfaces/ZDAIInterface.sol";
+
+// Contracts
+import "./Initializable.sol";
+
+
+contract DAIPool is DAIPoolInterface, Initializable {
+    using AddressLib for address;
+
+    /* State Variables */
+
+    IERC20 public dai;
+
+    ZDAIInterface public zdai;
+
+    LenderInfoInterface public lenderInfo;
+
+    LoansInterface public loanInfo;
+
+    /** Modifiers */
+
+    /* Constructor */
+
+    /** External Functions */
+
+    function initialize(
```

Comment:
`Initializable` and `DAIPool` both have different functions with the same name: `initialize`. I think it is best to name these different things. Line 65 is making a call to the same function name that it is within.

Reply:
Why it would be an issue? Contracts can have overloaded functions (https://solidity.readthedocs.io/en/v0.5.15/contracts.html#function-overloading) without problems.

Reply:
I know for a while truffle had an issue compiling overloaded names, but it may well be that that problem is now gone 👍